src/guide/essentials/forms.md

```diff
@@ -410,24 +412,26 @@ The `true-value` and `false-value` attributes don't affect the input's `value` a
 <input type="radio" v-model="pick" :value="second" />
 ```
 
-`pick` will be set to the value of `first` when the first radio input is checked, and set to the value of `second` when the second one is checked.
+Birinchi `radio` inputi checked bo'lganda, `first`ning qiymati `pick`ga tenglanadi va ikkinchisi checked bo'lganda, `second`ning qiymati `pick`ga tenglanadi.
 
-### Select Options
+### Select Variantlari
 
 ```vue-html
 <select v-model="selected">
-  <!-- inline object literal -->
+  <!-- obyekt literal - bu obyekt e'lon qilishning bir yo'li -->
+  <!-- ichki obyekt literal -->
   <option :value="{ number: 123 }">123</option>
 </select>
 ```
 
-`v-model` supports value bindings of non-string values as well! In the above example, when the option is selected, `selected` will be set to the object literal value of `{ number: 123 }`.
+`v-model` `string` bo'lmagan qiymatlarning (non-string values) qiymatlarini bog'lashni ham qo'llab-quvvatlaydi. Yuqoridagi misolda,
+variant tanlanganda, `{ number: 123 }` obyekt literalining qiymatiga tenglanadi.
 
-## Modifiers
+## O'zgartiruvchilar (Modifiers)
```

Comment:
Modifikatorlar degan yaxshiroq menimcha.

Reply:
To'g'rilandi.